packages/actions/src/Concerns/CanImportRecords.php

```diff
@@ -140,7 +142,8 @@ protected function setUp(): void
                         $csvReader->setDelimiter($csvDelimiter);
                     }
 
-                    $csvReader->setHeaderOffset(0);
+                    $headerOffset = $this->getHeaderOffset();
```

Comment:
When using the getter, please utilise `$action` instead of `$this`. For, reasons I don't want to explain lol.